File: Python/baekjoon/단계별로 풀어보기/조건문/2480_주사위 세개.py
# https://www.acmicpc.net/problem/2480

scores = sorted(map(int, input().split()))
if len(set(scores)) == 1:
    print(10000 + int(scores[0]) * 1000)
elif len(set(scores)) == 2:
    print(1000 + scores[1] * 100)
else:
    print(int(scores[-1]) * 100)

File: C#/Baekjoon/조건문/2480_주사위 세개.cs
//https://www.acmicpc.net/problem/2480
int[] n = Array.ConvertAll(Console.ReadLine().Split(), int.Parse);
Array.Sort(n);
if (n[0] == n[1] && n[1] == n[2])
{
    Console.WriteLine($"{10000 + 1000 * n[0]}");
}
else if (n[0] != n[1] && n[1] != n[2])
{
    Console.WriteLine($"{100 * n.Max()}");
}
else
{
    Console.WriteLine($"{1000 + 100 * (n[0] == n[1] ? n[0] : n[1])}");
}